File: UsuariosAPI/src/main/java/com/clinica/api/personal_service/repository/RolRepository.java
package com.clinica.api.personal_service.repository;

import com.clinica.api.personal_service.model.Rol;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface RolRepository extends JpaRepository<Rol, Long> {
}

File: usuario-service/src/main/kotlin/com/clinica/api/usuario_service/repository/RolRepository.kt
package com.clinica.api.usuario_service.repository

import com.clinica.api.usuario_service.model.Rol
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.Optional

@Repository
interface RolRepository : JpaRepository<Rol, Long> {
    fun findByNombre(nombre: String): Optional<Rol>
}
